asf_search/search/search.py

```diff
@@ -49,6 +49,9 @@ def search(
         relativeOrbit: Union[int, Tuple[int, int], Iterable[Union[int, Tuple[int, int]]]] = None,
         season: Tuple[int, int] = None,
         start: Union[datetime.datetime, str] = None,
+        absoluteBurstID: int = None,
+        relativeBurstID: int = None,
```

Comment:
Are these really ints? I thought there was more to the burst ID than just [0-9]. Would affect comments up above about `parse_int_list`, making them `parse_string_list`

Reply:
I'm pretty sure absolute and relative burst id are only ints (at least, I haven't run into any issues treating them as ints). Will have to ask and verify that this is the case.

Reply:
Just confirmed, relative and absolute burst IDs are integers